server/src/main/java/org/elasticsearch/cluster/coordination/Reconfigurator.java

```diff
@@ -148,6 +148,10 @@ public ClusterState maybeReconfigureAfterNewMasterIsElected(ClusterState cluster
         return clusterState;
     }
 
+    public void ensureVotingConfigCanBeModified() {
```

Comment:
Could we add a comment indicating that this is a workaround that we should remove once https://github.com/elastic/elasticsearch/issues/98055 is resolved?